BE/Trusts/Trusts.rdf

```diff
@@ -278,10 +260,46 @@
-		<skos:definition>Entity that establishes a trust and place property under the protection and management of one or more trustees for the immediate or eventual benefit of ascertainable one or more beneficiaries. It is not always necessary to identify the trustor who may be also be a trustee and/or one of the beneficiaries. In legal parlance, a trustor is called a settlor in the UK and a grantor in the US, whereas in common usage he or she may also be called a creator, donor, initiator, owner, or Trust maker.</skos:definition>
-		<fibo-fnd-utl-av:definitionOrigin rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/trustor.html</fibo-fnd-utl-av:definitionOrigin>
+		<skos:definition>party that establishes a trust and places property under the protection and management of one or more trustees for the benefit of at least one beneficiary</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom>Barron&apos;s Dictionary of Finance and Investment Terms, Ninth Edition, 2014.</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/trustor.html</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:explanatoryNote>It is not always necessary to identify the trustor who may be also be a trustee and/or one of the beneficiaries. In legal parlance, a trustor is called a settlor in the UK and a grantor in the US, whereas in common usage he or she may also be called a creator, donor, initiator, owner, or trust maker.</fibo-fnd-utl-av:explanatoryNote>
 		<fibo-fnd-utl-av:synonym>grantor</fibo-fnd-utl-av:synonym>
 		<fibo-fnd-utl-av:synonym>settlor</fibo-fnd-utl-av:synonym>
 	</owl:Class>
+	
+	<owl:ObjectProperty rdf:about="&fibo-be-tr-tr;hasBeneficiary">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;actsOn"/>
+		<rdfs:label>has beneficiary</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-be-tr-tr;Trust"/>
+		<rdfs:range rdf:resource="&fibo-be-tr-tr;TrustBeneficiary"/>
+		<owl:inverseOf rdf:resource="&fibo-be-tr-tr;isBeneficiaryOf"/>
+		<skos:definition>links a trust to a named beneficiary</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-be-tr-tr;hasTrustee">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;isAffectedBy"/>
+		<rdfs:label>has trustee</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-be-tr-tr;Trust"/>
+		<rdfs:range rdf:resource="&fibo-be-tr-tr;Trustee"/>
+		<owl:inverseOf rdf:resource="&fibo-be-tr-tr;isTrusteeOf"/>
+		<skos:definition>links a trust to a named trustee</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-be-tr-tr;isBeneficiaryOf">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;isAffectedBy"/>
+		<rdfs:label>is beneficiary of</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-be-tr-tr;TrustBeneficiary"/>
+		<rdfs:range rdf:resource="&fibo-be-tr-tr;Trust"/>
+		<skos:definition>specifies the trust that a beneficiary is named in</skos:definition>
```

Comment:
Why singular ("the trust")? Could someone not be a beneficiary of many trusts? There is no such restriction on Beneficiary.